cluster.go

```diff
@@ -767,11 +767,19 @@ func cmdSlot(cmd Cmder, pos int) int {
 	if pos == 0 {
 		return hashtag.RandomSlot()
 	}
-	firstKey := cmd.stringArg(pos)
-	return hashtag.Slot(firstKey)
+	val, ok := cmd.Args()[pos].(int)
+	if ok {
+		return val
+	}
+	return hashtag.Slot(cmd.stringArg(pos))
```

Comment:
Can we revert this change?

Reply:
So if you look at this line of code, https://github.com/go-redis/redis/blob/f3804cd0b74c6515ef206dabe7b314f2765fc788/cluster.go#L794

We reference the code that I changed. Would you suggest I change this method call to the cluster client specific method?

Reply:
I updated the code to reflect the change I was referring to.